packages/kit/src/types/ambient.d.ts

```diff
@@ -7,6 +7,7 @@
  * 		// interface Error {}
  * 		// interface Locals {}
  * 		// interface PageData {}
+ * 		// interface PageProps {}
```

Comment:
this doesn't belong here — `PageProps` isn't part of the `App` namespace, it's part of `./$types`

```suggestion
```